apps/sms/services/test/unit/mock_message_manager.js

```diff
@@ -27,7 +27,7 @@ var MockMessageManager = {
   retrieveMMS: function() {},
   markMessagesRead: function() {},
   markThreadRead: function() {},
-  findThreadFromNumber: function() {},
+  findThreadFromNumber: () => Promise.resolve(),
```

Comment:
not sure about this because the promise does not hold any value here; as a result it's as incorrect as before except issues will be more difficult to find (because .then() will work but the handler won't)